src/pkg/packager/creator/normal_test.go

```diff
@@ -56,3 +58,44 @@ func TestDifferentialPackagePathSetCorrectly(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadPackageDefinition(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		testDir   string
+		expectErr bool
+	}{
+		{
+			name:      "valid package definition",
+			testDir:   "valid",
+			expectErr: false,
```

Comment:
Would be nice to change this to expectedErr that is a string. That way you can do require.EqualError to see if the errors match. Alternatively do a require.ErrorContains.

Reply:
updated to check error string value